mac/Keyman4MacIM/Keyman4MacIM/KMInputController.m

```diff
@@ -174,9 +175,18 @@ - (void)menuAction:(id)sender {
     if (itag == 2) {
         // Using `showConfigurationWindow` instead of `showPreferences:` because `showPreferences:` is missing in
         // High Sierra (10.13.1 - 10.13.3). See: https://bugreport.apple.com/web/?problemID=35422518
-        // TODO: This bug should be fixed in 10.13.4, so after that has been available for a few weeks, we
-        // can create a PR for branch mac-revert-high-sierra-showPreferences-workaround and merge it.
-        [self.AppDelegate showConfigurationWindow];
+        // rrb: where available (everywhere but 10.13.1-10.13.3) call showPreferences, otherwise, call our workaround
+        u_int16_t systemVersion = [KMOSVersion SystemVersion];
+        if (0x0AD1 <= systemVersion && systemVersion <= 0x0AD3) // between 10.13.1 and 10.13.3 inclusive
```

Comment:
Can we use defined constants instead of magics please?

Reply:
```suggestion
        if (KMOSVersion.Version_10_13_1 <= systemVersion && systemVersion <= KMOSVersion.Version_10_13_3) // between 10.13.1 and 10.13.3 inclusive
```

Reply:
Named constants added and used, and the magic documented.